vllm/envs.py

```diff
@@ -213,6 +214,11 @@
     # Both spawn and fork work
     "VLLM_WORKER_MULTIPROC_METHOD":
     lambda: os.getenv("VLLM_WORKER_MULTIPROC_METHOD", "spawn"),
+
+    # Timeout for fetching images when serving multimodal models
+    # Default is 3 seconds
+    "VLLM_IMAGE_FETCH_TIMEOUT":
+    lambda: int(os.getenv("VLLM_IMAGE_FETCH_TIMEOUT", "3")),
```

Comment:
this is a bit too short for large image? maybe 5/10s?

Reply:
I will bring this to 5s for now, and add documentation that this can be configured by setting this env.